op-mode-definitions/show-isis.xml

```diff
@@ -0,0 +1,182 @@
+          </leafNode>
+          <node name="interface">
+            <properties>
+              <help>Show IS-IS interfaces</help>
+                <completionHelp>
+                  <script>${vyos_completion_dir}/list_interfaces.py</script>
+                </completionHelp>
+            </properties>
+            <children>
+              <leafNode name="detail">
+                <properties>
+                  <help>Show detailed information</help>
+                </properties>
+                <command>/usr/bin/vtysh -c "show isis interface detail"</command>
+              </leafNode>
+            </children>
+            <command>/usr/bin/vtysh -c "show isis interface"</command>
+          </node>
+          <tagNode name="interface">
+            <properties>
+              <help>Show specific IS-IS interface</help>
+                <completionHelp>
+                  <script>${vyos_completion_dir}/list_interfaces.py</script>
+                </completionHelp>
+            </properties>
+            <command>/usr/bin/vtysh -c "show isis interface $4"</command>
+          </tagNode>
+          <node name="traffic-engineering">
+            <properties>
+              <help>Show IS-IS MPLS traffic engineering information</help>
+            </properties>
+            <children>
+              <leafNode name="router">
+                <properties>
+                  <help>Show router information</help>
+                </properties>
+                <command>/usr/bin/vtysh -c "show isis mpls-te router"</command>
+              </leafNode>
+              <leafNode name="interface">
+                <properties>
+                  <help>Show interface information</help>
+                </properties>
+                <command>/usr/bin/vtysh -c "show isis mpls-te interface"</command>
+              </leafNode>
+              <tagNode name="interface">
+                <properties>
+                  <help>Show specific IS-IS interface</help>
+                    <completionHelp>
+                      <script>${vyos_completion_dir}/list_interfaces.py</script>
+                    </completionHelp>
+                </properties>
+                <command>/usr/bin/vtysh -c "show isis mpls-te interface $5"</command>
+              </tagNode>
+            </children>
+          </node>
+          <node name="neighbor">
+            <properties>
+              <help>Show IS-IS neighbor adjacencies</help>
+            </properties>
+            <children>
+              <leafNode name="detail">
+                <properties>
+                  <help>Show detailed information</help>
+                </properties>
+                <command>/usr/bin/vtysh -c "show isis neighbor detail"</command>
+              </leafNode>
+            </children>
+            <command>/usr/bin/vtysh -c "show isis neighbor"</command>
+          </node>
+          <tagNode name="neighbor">
+            <properties>
```

Comment:
Please add a completion helper list for system-id and detail.

Reply:
Will make change.

Reply:
This will result in a list of `system` and `id`, it should rather be `system-id` and `detail` as this is what FRR accepts.